server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -268,8 +266,6 @@ public void testExtensionsDiscovery() throws Exception {
     }
 
     public void testNonUniqueExtensionsDiscovery() throws Exception {
-        Path extensionDir = createTempDir();
```

Comment:
I think this one might need to stay as well as it overwrites the other tests.  Again, a different tempDir variable name would help resolve any confusion.